docs/src/content/en/docs/frameworks/express.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: "Getting started with Mastra and Express | Mastra Guides"
+description: A step-by-step guide to integrating Mastra with an Express backend.
+---
+
+import { Callout, Steps, Tabs, FileTree } from "nextra/components";
+
+# Integrate Mastra in your Express project
+
+Mastra integrates with Express, making it easy to:
+
+- Build flexible APIs to serve AI-powered features
+- Maintain full control over your server logic and routing
+- Scale your backend independently of your frontend
+
+Use this guide to scaffold and integrate Mastra with your Express project.
+
+<Callout type="warning">
+This setup is compatible with the following package versions:
+- `express`: 4.x
+- `@types/express`: 4.x
+
+Type compatibility in 5.x can be inconsistent while `express` and `@types/express` evolve toward alignment.
+</Callout>
+
+<Steps>
+## Install Mastra
+
+Install the required Mastra packages:
```

Comment:
I'm not 100% sure this is required, `npx mastra@latest init --default` will prompt to install the packages.